package.json

```diff
@@ -4,19 +4,20 @@
   "description": "Timesheet app using the SAP Cloud SDK and Express.js",
   "main": "src/index.ts",
   "scripts": {
+    "megacleaninstall": "rm -f package-lock.json && find . -name \"node_modules\" -exec rm -rf '{}' + && find db/src -name \"gen\" -exec rm -rf '{}' + && find . -name \"sqlite.db\" -exec rm -rf '{}' + && npm i",
```

Comment:
Really?

Reply:
Ask @jjtang1985 , I changed it to "clean"